modules/social_features/social_group/config/install/flag.flag.mute_group_notify.yml

```diff
@@ -0,0 +1,30 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - group
+id: mute_group_notify
+label: 'Mute Group Notify'
+bundles: {  }
+entity_type: group
+global: false
+weight: 0
+flag_short: 'Mute notify'
```

Comment:
'Mute'

Reply:
Moritz suggested calls it a "Mute group". So I renamed it "Mute group".